wiki/Guides/How_To_Get_Your_Map_Ranked/en.md

```diff
@@ -6,19 +6,19 @@ outdated: true
 
 *See also: [Ranking Criteria](/wiki/Ranking_Criteria)*
 
-Having trouble getting your [map](/wiki/Glossary) ranked? You've come to the right place.
+Having trouble getting your [beatmap](/wiki/Glossary) ranked? You've come to the right place.
 
 ## Step 1: Modding
 
 ***Precursor: [Getting Your Map Modded](/wiki/Modding_Guides/Getting_Your_Map_Modded)***
 
-The first and most important step towards getting your map ranked is to get people looking at it - people willing to [mod](/wiki/Modding) your map can be found in \#modreqs, the modding queues subforum or by asking them directly after checking their signature for modding policies. Remember that all mods are useful - any perspective on your map can be used to make it a better one. Consider every suggestion a modder makes, and if you turn any down, post a detailed response in the map thread making it clear the reasons why. It makes them a better modder and makes you think more about how your map can improve! Also, if you found the mod particularly useful, be sure to give the post a [kudosu](/wiki/Glossary).
+The first and most important step towards getting your map ranked is to get people looking at it - people willing to [mod](/wiki/Modding) your map can be found in \#modreqs, the [modding queues forum](https://osu.ppy.sh/community/forums/60) or by asking them directly after checking their signature for modding policies. Remember that all mods are useful - any perspective on your map can be used to make it a better one. Consider every suggestion a modder makes, and if you turn any down, post a detailed response in the map thread making it clear why. It makes them a better modder and makes you think about how your map can improve! Also, if you found the mod particularly useful, be sure to give the post a [kudosu](/wiki/Glossary).
 
-Your map's star priority is important when it comes to moving on to the next step. If a modder particularly likes your map they might give it a kudosu star - the more you work to make your map better, the more likely this will be. You can also gain stars by modding other people's maps and spending the kudos on your own mapset. If you don't think you're very good at modding, practice makes perfect! Even a few small suggestions can really help another mapper out, so don't be afraid to give modding a go.
+Your map's star priority is important when it comes to moving on to the next step. If a modder particularly likes your map they might give it a kudosu star - the more you work to make your map better, the more likely it will be. You can also gain stars by modding other people's maps and spending the kudosu stars on your own mapset. If you don't think you're very good at modding, practice makes perfect! Small suggestions can really help another mapper out, so don't be afraid to give modding a go.
```

Comment:
The term Kudosu is only from people upvoting your initial thread on a modding page, the points can not be used at all.

Reply:
Also, this entire paragraph is made redundant due to the moddingv2 system

Reply:
Rewrote the entire paragraph to convert to moddingV2